homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -187,8 +196,17 @@ def _calc_precipitation(rain, snow):
         return round(rain_value + snow_value, 1)
 
     @staticmethod
-    def _get_condition(weather_code):
+    def _get_condition(hass, weather_code, timestamp=None):
```

Comment:
remove `@staticmethod`, replace `hass` by `self`, then you easily can get hass object by calling `self.hass`

Reply:
Done